src/Expensify.js

```diff
@@ -131,8 +134,27 @@ class Expensify extends PureComponent {
         ActiveClientManager.init();
     }
 
+    showSplash() {
+        Log.info('[BootSplash] showing splash screen', false);
+        BootSplash.show({fade: true});
+    }
+
     hideSplash() {
-        BootSplash.hide({fade: true});
+        Log.info('[BootSplash] hiding splash screen', false);
+        BootSplash.hide({fade: true})
+            .catch(error => Log.alert('[BootSplash] hiding failed', {message: error.message, error}, false));
+    }
+
+    reportBootSplashStatus() {
+        BootSplash.getVisibilityStatus()
+            .then((status) => {
+                Log.info('[BootSplash] Splash screen status', false, {status});
+
+                if (status === 'visible') {
+                    const parameters = _.omit(this.props, 'children');
+                    Log.alert('[BootSplash] Still visible. Current <Expensify /> props', parameters, false);
```

Comment:
I think you could make this all into a single line, like so: 
```JS
Log.alert('[BootSplash] splash screen still visible', {props: _.omit(this.props, 'children')}, false);
```

Reply:
I was trying to avoid this: 
![image](https://user-images.githubusercontent.com/12156624/144082403-f421d7fd-a9ec-488f-b09f-8f887e242a2b.png)

There's an eslint warning for lines longer than 120 characters 
Also it's a bit easier to read when we're not calling a function within the brackets of another function

Reply:
I see, makes sense!